com.spotify.Client.json

```diff
@@ -150,6 +150,88 @@
                 }
             ]
         },
+        /* THESE CANNOT BE UPDATED UNTIL SPOTIFY SUPPORTS ffmpeg 5 ------------------------------------------------------- */
+        {
+            "name": "sdl2",
```

Comment:
This is in runtime

Reply:
You are correct it is. but the version is too new for ffmpeg 4.4 and so without it ffmpeg will not compile. this took me a minute to figure out

Reply:
Somewhat [Arch linux](https://archlinux.org/packages/extra/x86_64/ffmpeg4.4/) builds ffmpeg 4.4 with latest sdl

Reply:
Strange. I couldnt get it working. When i used the latest version the compiler would throw a warning about the version needing to be less than 2.1

Reply:
Could you add this patch to ffmpeg and drop sdl2? https://github.com/FFmpeg/FFmpeg/commit/e5163b1d34381a3319214a902ef1df923dd2eeba

Reply:
ill do this and test now

Reply:
ok we should be good now